Update note on ptrace (#31075)
dotnet/docs: docs/core/diagnostics/debug-linux-dumps.md

## Collect dumps on Linux

> [!NOTE] Frequently asked questions regarding dump collection, analysis and other caveats can be found [here](FAQ-dumps.md).

Reviewer: ```suggestion
> [!TIP]
> For frequently asked questions about dump collection, analysis, and other caveats, see [Dumps: FAQ](faq-dumps.md).
```

## Collect dumps on Linux

> > [!TIP] 

Reviewer: ```suggestion
> [!TIP]
```

> > For frequently asked questions about dump collection, analysis and other caveats, see [DUMPS: FAQ](faq-dumps.yml).

Reviewer: ```suggestion
> For frequently asked questions about dump collection, analysis, and other caveats, see [Dumps: FAQ](faq-dumps.yml).
```
